backend/data-access/instructorsData.js

```diff
@@ -0,0 +1,84 @@
+const fs = require('fs');
+const PATH = './data/instructors.json';
+
+/**
+ * Get all instructors from database
+ * 
+ * @returns {Array} a list of all instructors stored in BD
+ */
+const getInstructors = () => {
```

Comment:
All of these methods can be a part of the new `instructorsService.js`. No need for another file